vulture_os/gui/crontab/reconcile.py

```diff
@@ -42,36 +42,36 @@
 # Mongo import
 from toolkit.mongodb.mongo_base import MongoBase
 
+REDIS_LIST = "darwin_alerts"
+REDIS_CHANNEL = "darwin.alerts"
+REDIS_RECONCILIED_CHANNEL = "vlt.darwin.alerts"
```

Comment:
Move this declaration to the Django settings.py

Reply:
done :)